src/js/azdata/azdataQueryPlan.js

```diff
@@ -340,6 +340,9 @@ azdataQueryPlan.prototype.init = function (container, iconPaths) {
                     node: this.queryPlanGraph
                 }
             ];
+
+        var maxX = -1;
+        var maxY = -1;
         while (stack.length > 0) {
             var entry = stack.pop();
             if (entry.node.children) {
```

Comment:
the max calculations still work if the execution plan is a single node (i.e. without children)?  I forgot whether this condition is guaranteed to be true at least once for all inputs.

Reply:
Good catch. While I have never seen a plan with a single node but it does not hurt to handle that case over here.